Lombiq.HelpfulLibraries.AspNetCore/Localization/LocalizedHtmlStringExtensions.cs

```diff
@@ -11,17 +11,22 @@ public static class LocalizedHtmlStringExtensions
     /// Returns a raw HTML string that's been JSON serialized and therefore safe to use within <c>&lt;script&gt;</c>
     /// tags in a Razor view.
     /// </summary>
-    public static IHtmlContent Json(this LocalizedHtmlString htmlString)
+    public static IHtmlContent Json(this LocalizedHtmlString htmlString) =>
+        htmlString?.Html() is { } html
+            ? new HtmlString(JsonConvert.SerializeObject(
+                html,
+                new JsonSerializerSettings { StringEscapeHandling = StringEscapeHandling.EscapeHtml }))
+            : new HtmlString("null");
+
+    /// <summary>
+    /// Returns a raw HTML string that's not encoded.
```

Comment:
[This test class](https://github.com/dotnet/runtime/blob/main/src/libraries/System.Text.Encodings.Web/tests/HtmlEncoderTests.cs) (linked from [here](https://learn.microsoft.com/en-us/dotnet/api/system.text.encodings.web.htmlencoder?view=net-7.0)) implies that it actually does encode a bunch of characters.